plugins/node/opentelemetry-instrumentation-bunyan/src/bunyan.ts

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  context,
+  diag,
+  getSpan,
+  isSpanContextValid,
+  Span,
+} from '@opentelemetry/api';
+import {
+  InstrumentationBase,
+  InstrumentationNodeModuleDefinition,
+  isWrapped,
+  safeExecuteInTheMiddle,
+} from '@opentelemetry/instrumentation';
+import { BunyanInstrumentationConfig } from './types';
+import { VERSION } from './version';
+import type * as BunyanLogger from 'bunyan';
+
+export class BunyanInstrumentation extends InstrumentationBase<
+  typeof BunyanLogger
+> {
+  constructor(config: BunyanInstrumentationConfig = {}) {
+    super('@opentelemetry/instrumentation-bunyan', VERSION, config);
+  }
+
+  protected init() {
+    return [
+      new InstrumentationNodeModuleDefinition<typeof BunyanLogger>(
+        'bunyan',
+        ['*'],
```

Comment:
would be safer to use `<2` since they might change internals between majors version so the instrumentation doesn't break user applications in the future

Reply:
Added it, thanks